kibana.md

```diff
@@ -370,20 +370,44 @@ The solution to this is to reorder the ACL blocks, so the ones that authenticate
 
 When a user logs in, ReadonlyREST will write an encrypted cookie in the browser. This cookie has an time to live that can be tweaked with the following configuration key in `kibana.yml`.
 
-```
+```yaml
 readonlyrest_kbn.session_timeout_minutes: 600 # defaults to 4320 (3 days)
 ```
 
 #### Clearing Session History
 
 By default, all the session data like search history, dev tool commands history, etc, will be wiped out from the browser whenever a new user is logged in, or a user changes tenancy. To override this behaviour, use this setting:
 
-```
+```yaml
 readonlyrest_kbn.clearSessionOnEvents: ["never"]
 ```
 
 Possible values: `"login", "tenancyHop", "never"`.
 
+#### No authentication rule defined
+
+The “problem with the configuration of authentication” error message is presented in ReadonlyREST Free/PRO/Enterprise when the login request is checked by the ACL and gets accepted by an ACL block with no authentication rule in it.
+
+An example of this would be:
+
+```yaml
+readonlyrest:
+   access_control_rules:
+   - name: "LDAP Auth"
+     ldap_authentication: ...
+   
+   - name: "Allow requests from localhost"
```

Comment:
Let's also propose a solution to the example case: i.e. avoid using hosts rule in general, or at least using it alone without strictly limiting it with some other rules like actions.

Reply:
Added ✔️